django-stubs/db/migrations/operations/models.pyi

```diff
@@ -107,6 +107,19 @@ class RemoveIndex(IndexOperation):
     name: str
     def __init__(self, model_name: str, name: str) -> None: ...
 
+class RenameIndex(IndexOperation):
+    model_name: str
+    new_name: str
+    old_name: str | None
+    old_fields: Sequence[str] | None
+    def __init__(
+        self,
+        model_name: str,
+        new_name: str,
+        old_name: str | None = ...,
+        old_fields: Sequence[str] | None = ...,
+    ) -> None: ...
```

Comment:
Looks like there are also two `cached_property`ies: old_name_lower, new_name_lower.

I doubt anyone wants to use those, but still best to be complete :)

Reply:
Thanks for pointing those out. I completely missed them, but are now added.